.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: snyk
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - '**'     # matches every branch
+      - '!main'  # excludes main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  security:
+    runs-on:
+      - self-hosted
```

Comment:
Please change to ubuntu-latest as its not a private repo and also github runners will be able to run